nixops_encrypted_links/plugin.py

```diff
@@ -38,11 +38,7 @@ def nixexprs():
 
     @staticmethod
     def load():
-        return [
-            "nixops_aws.resources",
-            "nixops_aws.backends.ec2",
-            "nixops_aws.resources.ec2_keypair",
-        ]
+        return []
```

Comment:
Not required -- it works with nixops-{aws,packet,vbox} without having to have any load hooks here.